src/Apps/Search/routes.tsx

```diff
@@ -29,8 +29,13 @@ const entityTabs = Object.entries(tabsToEntitiesMap).map(([key, entities]) => {
   return {
     path: key,
     Component: SearchResultsEntityRoute,
+
+    // FIXME: We shouldn't overwrite our route functionality, as that breaks
+    // global route configuration behavior.
```

Comment:
I'm trying to follow, but a little more context here would be good. So, providing a `render` method here technically overwrites the route functionality and can break things?

Reply:
Definiting a local `render` call on a route definition overwrites the handlers inside of Route.tsx, via a callback. We overwrite all over; the only reason I added this here was because soon I was going to merge this new app into the top level router and would like to return to this behavior.